test/undo.test.js

```diff
@@ -200,6 +200,83 @@ describe('Undo', () => {
     });
   });
 
+
+  describe('Undo/redo events fired with custom shortcuts', () => {
+    let undo;
+
+    beforeEach(() => {
+      const { config } = tools.undo;
+      undo = new Undo({ editor, config });
+      undo.initialize(initialData.blocks);
+      undo.save(firstChange.blocks);
+      undo.save(secondChange.blocks);
+    });
+
+    it('undo event inside Editor\'s holder has to cause changes in Undo Plugin stack', () => {
```

Comment:
Depending on the code standarts ruleset chosen in the project, this line could be transformed into `"undo event inside Editor's holder has to cause changes in Undo Plugin stack"`, because it looks cleaner. But it's just a comment.